packages/office-ui-fabric-react/src/utilities/dragdrop/DragDropHelper.tsx

```diff
@@ -285,8 +304,7 @@ export class DragDropHelper implements IDragDropHelper {
         let xDiff = this._dragData.clientX - event.clientX;
         let yDiff = this._dragData.clientY - event.clientY;
         if (xDiff * xDiff + yDiff * yDiff >= DISTANCE_FOR_DRAG_SQUARED) {
-          if (this._dragData.dragTarget &&
-            this._selection.isIndexSelected(options.selectionIndex)) {
```

Comment:
Removing the selection check changes an expectation regarding the behavior. Right now starting a drag on an unselected item can create a selection marquee, or allow for a pan. If we remove the selection check, we'll need another way to allow the host component to control when a drag/drop operation may start.